### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Moukarnold**
i  create a page language 
- i stored the language under the name storedlanguage
- storedlanguage will be used later with the router.push(.....) to passing away as data for the API/ Fetch
- english is setted as default language

### CODE REVIEW

**Comment by Moukarnold** [comment]
> i create a page language.
> 
> * i stored the language under the name storedlanguage
> * storedlanguage will be used later with the router.push(.....) to passing away as data for the API/ Fetch
> * english is setted as default language

File: pages/languages.js [hunk 1]
```diff
@@ -1,69 +1,90 @@
...
+    const storedLanguage = localStorage.getItem("language");
+    console.log("the lang is " + storedLanguage);
   }
 
-  console.log(languages);
+  useEffect(() => {
+    if (language === "") {
+      setLanguage("english");
+    }
+  }, [language]);
```
Comment (djfarly) [inline pages/languages.js]: Please do not roll your own local storage state implementation. Connecting state to local storage correctly is very hard and can lead to strange and hard to find bugs.

Please use the `use-local-storage-state` package we also used in the sessions. Check the the handouts for React with Local Storage to find out how to use it.

Basically all the code here can be simplified to:

```js
const [language, setLanguage] = useLocalStorageState("language", { defaultValue: "english" });
```

File: pages/Level.js [hunk 2]
```diff
@@ -1,43 +1,64 @@
...
+export default function PageLevel() {
+  const [levels, setLevels] = useState("Basic");
 
-export default function pageLanguage() {
-  function handleCheckEnglish() {}
-  function handleCheckFrench() {}
-  function handleCheckDeutsch() {}
+  function handleCheckLevel(level) {
+    setLevels(level);
+    console.log(level);
```
Comment (djfarly) [inline pages/Level.js]: this console.log does not want to be merged :)

```suggestion
```

File: pages/Options.js [hunk 3]
```diff
@@ -1,43 +1,67 @@
...
-  Box,
   InnereBox,
   SpaceLabel,
-} from "@/components/styles/container.styled";
+} from "@/components/styledComponents/Container.styled";
+
+import Link from "next/link";
+import { useState } from "react";
+
+const spk = "simple Sentences and for kids";
```
Comment (djfarly) [inline pages/Options.js]: It's strange that you have put this one option value into a variable while is directly in the JSX. Please consider having all options in an array and then mapping over that array in render.

Checkboxes are not the best way to represent a selection of one item. Please use a `<select>` element or radio buttons.



```js
const optionItems = [
  { label: "Simple Sentences & For Kids", value: "simple sentences and for kids"  },
  …
];
```

in the JSX:

```js
  <select name="options" onChange={(event) => handleOption(event.target.value)}>
    {optionItems.map((item) => (
      <option
        selected={options === item.value}
        value={item.value}
        key={item.value}
      >
        {item.label}
      </option>
    ))}
  </select>
```

File: pages/Options.js [hunk 4]
```diff
@@ -1,43 +1,67 @@
...
+export default function PageOptions() {
+  const [options, setOptions] = useState(spk);
 
-export default function pageLanguage() {
-  function handleCheckEnglish() {}
-  function handleCheckFrench() {}
-  function handleCheckDeutsch() {}
+  function handleOption(option) {
+    setOptions(option);
+    console.log(option);
```
Comment (djfarly) [inline pages/Options.js]: snip
```suggestion
```

File: pages/languages.js [hunk 5]
```diff
@@ -1,69 +1,90 @@
...
+                checked={language === "deutsch"}
+                name="deutsch"
+                value="deutsch"
+              />{" "}
+              🇩🇪
+              <SpaceLabel>
+                <h3>Deutsch</h3>
+              </SpaceLabel>
+            </ContainerMainLabel>
+          </form>
```
Comment (djfarly) [inline pages/languages.js]: See my comment above. Please us a select or radio buttons to represent this input.

File: pages/Level.js [hunk 6]
```diff
@@ -1,43 +1,64 @@
...
+              checked={levels === "Advanced"}
+              value="Advanced"
+              name="Advanced"
+            />
+            <SpaceLabel>
               {" "}
               <h3>Advanced</h3>{" "}
-            </span>
+            </SpaceLabel>
           </ContainerMainLabel>{" "}
```
Comment (djfarly) [inline pages/Level.js]: this should be a select or radiogroup as well :)

**Review by djfarly — CHANGES_REQUESTED** [review]
Hej Arnold,

thank you for the PR!

Your app consists mostly of unconnected states and empty pages. For future PRs and stories please make sure that you are developing and actual feature.

You are representing all you selectable states as checkboxes. Please use a `<select>` element or a radio group to do so instead. (See comments below).

Keeping going!

**Review by antoniatandler — COMMENTED** [review]
I think this will work well! 😊

**Pull request merged**